lib/gds_api/content_store.rb

```diff
@@ -18,9 +20,85 @@ def content_item!(_)
     raise "`ContentStore#content_item!` is deprecated. Use `ContentStore#content_item` instead"
   end
 
+  def self.redirect_for_path(content_item, request_path, request_query = "")
+    RedirectResolver.call(content_item, request_path, request_query)
+  end
+
 private
 
   def content_item_url(base_path)
     "#{endpoint}/content#{base_path}"
   end
+
+  class RedirectResolver
+    def initialize(content_item, request_path, request_query)
+      @content_item = content_item
+      @request_path = request_path
+      @request_query = request_query
+    end
+
+    def self.call(content_item, request_path, request_query = "")
```

Comment:
I would use `*args` here to avoid having to update this is the args to `initialize` change.

Reply:
Oh cool, does that work for all argument types? I had it as just for hashes in my head. Will give it a spin.